augur/data/schema-measurements.json

```diff
@@ -89,8 +89,18 @@
                         "description": "The short label to display for the x-axis that describes the `value` of the measurements in a collection",
                         "type": "string"
                     },
+                    "thresholds": {
+                        "description": "Numeric measurement thresholds to be displayed as solid grey lines across subplots. Optional -- if not provided, no threshold will be displayed.",
+                        "type": "array",
+                        "minItems": 1,
+                        "items": {
+                            "description": "A numeric measurement threshold value.",
+                            "type": "number"
+                        }
+                    },
                     "threshold": {
-                        "description": "A numeric measurement threshold to be displayed as a single grey line shared across subplots. Optional -- if not provided, no threshold will be displayed",
+                        "deprecated": true,
+                        "description": "DEPRECATED - A numeric measurement threshold to be displayed as a single grey line shared across subplots. Will be ignored if 'thresholds' is provided.",
```

Comment:
Instead of deprecating one key and adding another, did you consider making the existing `threshold` key take _either_ a single number or an array of numbers? This doesn't solve the compatibility issue with Auspice, but it does simplify the schema a bit.

I guess it does mean that Auspice may barf on `threshold: […]` instead of merely ignoring `thresholds: […]`. Not entirely sure how gracefully it'd handle that type change, but I think it'd work out because of the [type test for whether to display the toggle or not](https://github.com/nextstrain/auspice/blob/c373bb21a45666120d34f158a18780e7802d84e4/src/components/controls/measurementsOptions.js#L115) and that [x1/x2 for the SVG line would get set to NaN](https://github.com/nextstrain/auspice/blob/c373bb21a45666120d34f158a18780e7802d84e4/src/components/measurements/measurementsD3.js#L158-L164).

Reply:
(from [another comment](https://github.com/nextstrain/augur/pull/1148#discussion_r1096104402))
> My thought here is that we will want to drop the deprecated threshold key as some point

Ah, my thinking was that we wouldn't have to drop support for a single int and would indefinitely accept both a single value and list of values, e.g., `threshold: x` and `threshold: [x, y, z]`.

Per [my other comment](https://github.com/nextstrain/augur/pull/1148#discussion_r1095244233), Augur would then continue to produce `threshold: x` when only a single value is given.

This combination maximizes compatibility without much effort on our part.

IMO, the "deprecation" of `threshold` vs. `thresholds` is unfortunate, because it's not really a deprecation, it's a removal: after version X, Augur won't produce it and thus if you have newer Augur and older Auspice, you're stuck on the Augur version prior to X.